develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -1,113 +1,61 @@
 # Development Reference [](id=development-reference)
 
 <p style="margin-left: 40px;">
-Here you'll find Liferay's reference documentation for Java, tag libraries,
+This page gives you access to reference documentation for Java, tag libraries,
 JavaScript and CSS, and deployment descriptors.
 </p>
 
 <p style="margin-left: 40px;">
 Liferay Portal 7 Community Edition's Portal Core reference documentation is
 available online  at <a href="https://docs.liferay.com/portal/7.0"
-target="_blank">https://docs.liferay.com/portal/<span
+target="_blank">https://docs.liferay.com/portal/7.0<span
```

Comment:
Do we really need to provide this generic URL at the beginning? In which cases is it a good idea for a developer to go there? Don't we provide more specific and relevant links in this article?